code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -83,6 +89,17 @@
 		add_blood(blood_color)
 	return
 
+/obj/item/weapon/melee/telebaton/proc/stun(mob/living/carbon/human/H, mob/living/user)
+	if(H.check_shields(src, 0, "[user]'s [name]"))
+		return FALSE
+	user.visible_message(SPAN_DANGER("[user] knocks down [H] with \the [src]!"),\
+							SPAN_WARNING("You knock down [H] with \the [src]!"))
+	// Visuals and sound
+	playsound(H, 'sound/weapons/Genhit.ogg', 75, TRUE, 7)
```

Comment:
sound is very loud and sounds a bit out of place

also there is no attack animation of the user hitting the target